app/prism/PrismLogic.scala

```diff
@@ -99,4 +101,26 @@ object PrismLogic {
       }
     }.getOrElse(true)
   }
+
+  /**
+    * @return Percentiles of instance AMIs age
+    */
+  def instancesAmisAgePercentiles(amis: List[AMI]): Percentiles = {
+    val ages = amis.flatMap { ami =>
+      val amiAge = ami.creationDate.map(DateUtils.daysAgo).getOrElse(0)
+      List.fill(ami.instances.getOrElse(Nil).length)(amiAge)
+    }
+    Percentiles(ages)
+  }
+
+  /**
+    * @return the number of instances from the list of AMIs that match a given SSA
+    */
+  def instancesCountForSSA(amis: List[AMI], ssa: SSA): Int = {
```

Comment:
There's nothing in the signature that says these amis have to have been "hydrated" with their instances, which seems dangerous. If this still seems right then perhaps that's worth a comment.

Also, is this used?